common/maven-url-handler/pom.xml

```diff
@@ -25,7 +25,7 @@
         <artifactId>helidon-build-common-project</artifactId>
         <version>3.0.0-SNAPSHOT</version>
     </parent>
-    <artifactId>helidon-archetype-maven-url-handler</artifactId>
+    <artifactId>helidon-maven-url-handler</artifactId>
```

Comment:
Looking at other modules under common, the convention is to use `helidon-build-common-` as an artifactId prefix.